src/actions/layers/Source.ts

```diff
@@ -0,0 +1,15 @@
+import TransformableImage from "../../transformation/TransformableImage";
```

Comment:
File name should match class name no?

Reply:
Source is not an Entrypoint in itself (Something we might want to discuss)

It's not possible to `import Source from 'base/actions/source'`

instead, it's imported through Layers

It's not possible to `import {Source} from 'base/actions/layers'.

A source on its own is not really an action, so I found it weird having it as an entrypoint.

Reply:
the class ImageSource exists in a file named Source. I would put it in ImageSource.ts file.
unless you want to export ImageSource, TextSource, etc from the Source.ts file

Reply:
For now, I think  that's how it's going to look, I have a feeling that we'll remove Source from the current place any way due to source being used in other, non `overlay` specific contexts

Such as this:

`.effect(Effect.styleTransfer(Source.image('lighthouse')`

I'm honestly not too sure where to place it right now.

Reply:
We need a Params folder

Reply:
or under Overlay